### DISCUSSION TITLE
Link images to media library in field type "image"

### DISCUSSION THREAD

**Discussion by kopfnick** [pods-framework/pods#6723] [Ideas]
When using the field type "image", one or multiple thumbnails with the image title get added to the editing interface. As soon as the thumbnails are there, there is no way to edit the images by going to the media library directly from that interface. 

Expected behavior would be to click on a thumbnail, this opens the media library with the relevant images "preselected", i can then edit the fields (for example), save, close and the changes are reflected immediately.

At the moment, this can only be done by going to the media library and searching for the images there, which can be a drag considering the state of WP media handling...

**Comment by JoryHogeveen** [comment]
Hi @kopfnick 
This is one of the features that will be introduced in Pods 2.7!

**Comment by pglewis** [comment]
Currently only pick fields are setup for modal add/edit but that's a technical detail.  

The expectation is that the edit icon for an image in a file field will edit modally just like pick relationships do, especially now that file upload and List View are nearly wearing the exact same clothes.  Any snags you see for enabling modal media edits on the PHP side, @sc0ttkclark?
